addons/sys_core/stringtable.xml

```diff
@@ -126,5 +126,25 @@
             <Japanese>ヘッドセットのトグル/スペクテイターのトグル</Japanese>
             <Italian>Attiva/Disattiva Cuffia - Attiva/Disattiva Spettatori</Italian>
         </Key>
+        <Key ID="STR_ACRE_sys_core_difficultyInterference_displayName">
+            <English>Model Interference?</English>
```

Comment:
CBA settings are accessible from within Eden and are saved in the mission, there should be no down-side to it at all.